README.md

```diff
@@ -36,3 +36,41 @@ dependencies {
   compile "com.teamwizardry.librarianlib:librarianlib-1.12:4.0-SNAPSHOT:deobf"
 }
 ```
+
+## Features
+(note: this list was thrown together based on the package list, so it is not exhaustive)
+
+- reflection-based animation
+- annotation-based automatic block/item annotation
+- base capability implementation
+- base blocks/items/entities/fluids/multiparts 
+- base tile entities with automatic serialization
+- automatic model generation for everything under the sun (plants, fences, doors, you name it)
+- per-chunk data storage
+- per-world data storage
+- annotation-based config handler
+- a base container that simplifies shift-clicking to just be `transferRule().from(inventory1).to(inventory2).filter { some item filter }`
+- a simplified event bus for liblib's gui framework
+- a giant gui framework (from what I can tell, it's one of, if not the, most advanced ones in MC)
+- a data-driven guide book implementation using said gui framework
+- a massively reworked version of the giant gui framework
```

Comment:
So, we got two? but old one is deprecated

Reply:
@Bluexin Yes but the book is very complicated and should stay for now. Maybe I can redo it at a later date or something